### PR TITLE
Update rke, cli, ui tags for 2.5.2-rc1

### PR DESCRIPTION
**Pull request by mrajashree**

### CODE REVIEW

File: pkg/apis/go.mod [hunk 1]
```diff
@@ -11,7 +11,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/rancher/eks-operator v1.0.1
 	github.com/rancher/norman v0.0.0-20200930000340-693d65aaffe3
-	github.com/rancher/rke v1.2.0
+	github.com/rancher/rke v1.2.2-rc1
```
Comment (cbron) [inline pkg/apis/go.mod:14]: No go.sum changes ?

**Review by cbron — COMMENTED** [review]
Once this merges we can bump it in master too. Thats not part of release checklist but good to keep them somewhat inline.

File: pkg/apis/go.mod (hunk 1 above)
Comment (mrajashree) [reply pkg/apis/go.mod:14]: there are go.sum changes too=> https://github.com/rancher/rancher/pull/29578/files#diff-3295df7234525439d778f1b282d146a4f1ff6b415248aaac074e8042d9f42d63R1029

Comment (cbron) [reply pkg/apis/go.mod:14]: I mean in `pkg/apis/`

Comment (mrajashree) [reply pkg/apis/go.mod:14]: ah, thanks!

**Review by cbron — APPROVED** [review]

**Pull request merged**